attending/index.py

```diff
@@ -0,0 +1,19 @@
+import csv
+from pathlib import Path
+import os
+
+
+def load_index(file: Path):
+    project_index = {}
+    assert file.is_file()
+    with open(str(file), 'r') as index:
+        index_reader = csv.reader(index)
+        for row in index_reader:
```

Comment:
How about unpacking this as 
```suggestion
        for module,project,url in url_reader:
            project_url[module] = url
            project_url[project] = url
```